apigentools/commands/generate.py

```diff
@@ -382,5 +382,13 @@ def pull_repository(self, language, branch=None):
                     cwd=output_dir,
                 )
             except subprocess.CalledProcessError:
-                log.error(f"{self.args.is_ancestor} is not ancestor of branch {branch}")
-                raise
+                log.warning(
+                    f"{self.args.is_ancestor} is not ancestor of branch {branch}"
```

Comment:
Small nit, but can we add to the log that we're going to modify the branch being pulled? Maybe something like:
```suggestion
                    f"{self.args.is_ancestor} is not ancestor of branch {branch}, attempting to update branch"
```